src/screens/Asset/ReceiveModal.js

```diff
@@ -124,7 +124,9 @@ export default class ReceiveModal extends React.Component<Props, *> {
         <ContentWrapper forceInset={{ top: 'never', bottom: 'always' }}>
           <WarningBanner rounded small />
           <QRCodeWrapper>
-            <WalletAddress>{address}</WalletAddress>
+            <WalletAddress onPress={() => Clipboard.setString(address)}>
```

Comment:
It's not clear when text has been copied - no info is shown to the user. Maybe we can add a toast?
(And it would also be nice to have a visual touch representation here)

Reply:
i can just add a copy symbol but it might jump over a line

Reply:
Made font smaller and added clipboard symbol

<img src="https://user-images.githubusercontent.com/37015091/80386610-e4eee980-88a7-11ea-9f11-010fae7442f7.png" width="33%" height="33%" />

Reply:
Actually changed the symbol and reverted the address. Since users are expected to tap to select the string, they'll be notified that the address has been copied successfully and they'll know no more effort is needed to get the address :)